website/screens/components/heading/specs/HeadingSpecsPage.tsx

```diff
@@ -80,7 +80,9 @@ const sections = [
                       <Code>text-transform</Code>
                     </td>
                     <td>initial</td>
-                    <td></td>
```

Comment:
Yes, you are right. It is not defined because the value in the specifications is the default value in CSS. I don't know if it is necessary to specify it in the specifications then.

Reply:
I would not specify it then. I think we are not specifying it in other components.

Reply:
Done!